### ISSUE TITLE
Cannot save dropout layer from a tensorflow model with tf2onnx.convert.from_keras (and load it with onnx)

### ISSUE THREAD

**Issue by glm-barcelin** [onnx/tensorflow-onnx#1845]
**Describe the bug**
I want to save a model that I trained with tensorflow and that contains Dropout layers. I need these layers at inference to use them in training_mode in order to measure the epistemic uncertainty of my model. I think the tf2onnx `convert.from_keras` function does not save the dropout layer. 

**Urgency**
none.

**System information**
- Windows 11
- Tensorflow Version: 2.7.0
- Python version: 3.8.10

**To Reproduce**
Here is the code example that I use: 

          import tf2onnx
          import onnx
          from tensorflow.keras.layers import ( Dense, Input, Dropout,)
          from tensorflow.keras.models import Model
          
          input_layer = Input(shape=(96))
          h = Dense(128, activation="relu")(input_layer)
          h = Dropout(0.1)(h)
          h = Dense(128, activation="relu")(h)
          h = Dropout(0.1)(h)
          h = Dense(12, activation=None)(h)
          
          model = Model(input_layer, h, name=name)
          
          model_onnx, _ =tf2onnx.convert.from_keras(model, output_path='test_dropout.onnx')
          
          test = onnx.load('test_dropout.onnx')
          test.graph.node

In the node displayed I have no mention of dropout. The thing is, I want to use the dropout layer in training_mode during inference to use MC dropout to have a measurement of the epistemic uncertainty. 

Is there something I am doing wrong here ? Or is it just no possible to do so ?

**Comment by fatcat-z** [comment]
During the conversion, those training related op will be removed by freezing the tf graph. So this behavior is expected.

**Comment by glm-barcelin** [comment]
So you're saying it is impossible to use Dropout in training mode at inference ? I need it to estimate the epistemic uncertainty of my neural network using Monte-Carlo Dropout.

**Comment by fatcat-z** [comment]
> So you're saying it is impossible to use Dropout in training mode at inference ? I need it to estimate the epistemic uncertainty of my neural network using Monte-Carlo Dropout.

You are right, dropout op won't exist in the final graph for inference.

**Comment by glm-barcelin** [comment]
Thanks a lot for your answer.
